src/main/java/core/basesyntax/service/RegistrationException.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+public class RegistrationException extends Exception {
```

Comment:
Good job. Have some rectification.
It is possible to narrow the extension to RuntimeException instead of Exception